### ISSUE TITLE
Is ONE valid syntax?

### ISSUE THREAD

**Issue by hanjoosten** [AmpersandTarski/Ampersand#509]
In the past, a construction like this was valid:

```
INTERFACE Overview : I[ONE]
  BOX [ as : V[ONE*A]
        BOX [ "id" : I
            , "r"  : r
            , s    : s
            , t    : t
            ]
      ]
```

Currently, we get the following runtime error:
![image](https://cloud.githubusercontent.com/assets/1159672/17888159/ef80f0d8-6929-11e6-8f6b-d4359abe9a81.png)

This is wrong. Either this synax should not be allowed, or there should not be a runtime error. What should be the case?

**Comment by hanjoosten** [comment]
By the way: It seems that the value searched for in ONE is the value of the current session.

**Comment by RieksJ** [comment]
With the design of the new frontend, we (@Michiel-s, me) decided that `_SESSION` and `ONE` would refer to the same atom. In the old frontend, `ONE` was used as a starting atom, which is basically the role that `_SESSION` has today. If that's all there is to this, then we might declare `ONE` to be obsolete. 

However, I remember that @stefjoosten said something that would make `ONE` useful from an RA point of view, but I forgot what. 

Personally, I could use `ONE` to refer to the current session atom that would not make `INTERFACE`s appear in the menu bar, e.g. `INTERFACE "myifc": V[ONE*SESSION];'_SESSION';...` would not show up in the menu bar.

**Comment by sjcjoosten** [comment]
Last scrum session we discussed this issue:

ONE is valid syntax, and the expected behavior is to have an interface similar to what would happen if a user-concept like "Persoon" were used in its place. There is only one atom in "ONE", and it could be displayed.

In this particular example, a list of all "A"'s would be displayed (due to the use of V).

**Comment by Michiel-s** [comment]
I'll fix this asap

**Comment by Michiel-s** [comment]
Fixed in #781
